dnf/cli/output.py

```diff
@@ -82,7 +83,10 @@ def tsi_cmp_key(tsi):
                     b.installed.append(tsi)
                     continue
                 elif reason == 'group':
-                    b.installed_group.append(tsi)
+                    if str(tsi.installed) in goal.module_members:
```

Comment:
Make sure this works on py2 and py3. (possible unicode issue)

Reply:
works with py3

Reply:
also with py2